client/app.tsx

```diff
@@ -4,12 +4,7 @@ import './reset.css';
 import './light-theme.css';
 import './dark-theme.css';
 import './global.css';
-// import MainPage from './components/MainPage/MainPage.js';
-import fooListFn from './components/foo-list';
 import catListFn from './components/cat-list';
-// import jobListFn from './components/job-list';
-
-import dashboardFn from './components/dashboard';
 import Layout from './layout';
 import {
```

Comment:
good job getting rid of code that doesn't need to be there. Makes the code look more clean and organized.